### DISCUSSION TITLE
Problem saving an global variable as offset

### DISCUSSION THREAD

**Discussion by LilTeo14** [EdgeTX/edgetx#2235] [Q&A] [answered]
Hello everyone i hope find solution to this problem.
I have a tx16s running edge tx.
The problem occurs when i try to save in channel 1  offset in mix tab as global variable 1.
![20220825_145236](https://user-images.githubusercontent.com/44703219/186812445-813b27a1-1ed7-4e4e-8c9a-d731d27e2f87.jpg)
I can set it succesfully it and it works perfectly, but every time i turn off the radio when i turn it on again, the offset is again set to 0.
![20220825_145253](https://user-images.githubusercontent.com/44703219/186812545-e014bf5d-21e6-4631-87d5-871aed8183fb.jpg)
And it dont let me save any change anymore, if i do any other change to the model they are not saved too.
Also when i try to read the sd from edge tx it shows me an error.
![Screenshot 2022-08-25 151011](https://user-images.githubusercontent.com/44703219/186812751-012fc9ba-f69b-4b1b-97ef-879774979ce4.png)

I want to clarify when i reset the radio and erase sd card content, it works fine again and i can save changes, but if i save the offset as global variable 1 it fail again and it dont let me to save changes anymore.

**Comment by pfeerick** [comment, marked as answer]
What version of EdgeTX? This sounds somewhat like an issue that was present during the 2.7 release candidates and was fixed by 2.7.0 - i.e. https://github.com/EdgeTX/edgetx/issues/1734. There were also some issues present in 2.6 around GVs as offsets, but I think that was more relating to importing from OpenTX bin files. However, I cannot repeat this using the current release version, 2.7.1. 

To be able to diagnose why `model1.yml` has an issue, you'd need to upload that file (or preferably zip the whole MODELS and RADIO directories in case there are other models with issues as well). Otherwise, you can usually open that model in the radio, toggle a setting on and off, and it will re-write the file, and clear the corrupt setting. Assuming, that is, you're not running a firmware build that writes the setting in a buggy fashion in the first place.

**Reply by LilTeo14** [reply]
Thank you. The radio was running 2.6 software, i updated it and everything works fine now.

**Comment by Eldenroot** [comment]
Better to report only issues with the latest stable release on board.,, :)
